invenio_damap/templates/semantic-ui/invenio_damap/macros/detail.html

```diff
@@ -9,10 +9,18 @@
 
 {%- macro render_dmp_button() %}
   {% set user_jwt = create_auth_jwt() %}
-  {% set dmp_list = query_damap_madmps(user_jwt)['hits']['hits'] if user_jwt is not none else [] %}
+  {% set dmp_list = query_damap_madmps(user_jwt) if user_jwt is not none else none %}
```

Comment:
nit: for easier python readability
```suggestion
  {% set dmp_list = query_damap_madmps(user_jwt) if user_jwt is not None else None %}
```

Reply:
This won't work. See [here](https://stackoverflow.com/a/19616065).

Reply:
Oh thanks for pointing it out :)